src/util.js

```diff
@@ -168,3 +168,26 @@ function ComparativeDate(referenceDate, date, options) {
     // TODO isPrev, isNext
   }
 }
+
+export function getDaysEvents(day, events) {
+  if (!events || events.length === 0) {
+    return [];
+  }
+
+  const result = [];
+    
+  for (const event of events) {
+    const { date, date2 } = event,
```

Comment:
What do you think about having `date2` be called `endDate`? Or even enforcing a type similar to:
```ts
{ date: DateLike } | { startDate: DateLike, endDate: DateLike } 
```
??